haiku-apps/beshare/beshare-2.32.recipe

```diff
@@ -36,27 +35,26 @@ REQUIRES="
 	lib:libsanta$secondaryArchSuffix
 	lib:libz
 	"
+
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libsanta$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-	#devel:muscle$secondaryArchSuffix
+#	devel:muscle$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
 	cmd:make
+	cmd:wget
 	makefile_engine
 	"
 
-PATCHES="beshare-$portVersion.patchset"
```

Comment:
@fbrosson yes, it works fine, and yeah you're right it should being removed, look like the patch just applied in another commit ex:
https://github.com/HaikuArchives/BeShare/blob/master/source/ChatWindow.cpp#L1175
https://github.com/HaikuArchives/BeShare/blob/master/source/ChatWindow.cpp#L14

But iam not sure too, since not all of them applied, just some.